src/uu/fmt/src/fmt.rs

```diff
@@ -155,7 +169,10 @@ fn parse_arguments(args: impl uucore::Args) -> UResult<(Vec<String>, FmtOptions)
             }
         };
         if !matches.get_flag(OPT_WIDTH) {
```

Comment:
Unrelated to your change, I'm getting a runtime error here because `OPT_WIDTH` is not marked as a flag in `clap`:
```
thread 'main' panicked at 'Mismatch between definition and access of `width`. Could not downcast to bool, need to downcast to alloc::string::String
', /home/terts/Programming/uutils/coreutils/src/uu/fmt/src/fmt.rs:171:21
```